src/components/views/DownloadModModal.vue

```diff
@@ -4,11 +4,18 @@
             <div class="modal-background" @click="setIsModProgressModalOpen(false);"></div>
             <div class='modal-content'>
                 <div class='notification is-info'>
-                    <h3 class='title'>Downloading {{$store.getters['download/currentDownload'].modName}}</h3>
-                    <p>{{Math.floor($store.getters['download/currentDownload'].progress)}}% complete</p>
+                    <h3 class='title'>Downloading and installing {{$store.getters['download/currentDownload'].modName}}</h3>
```

Comment:
I think having this this way was my earlier suggestion but now that the installation actually updates the modName this seems a bit awkward as installation starts to loop the same mod names again. This could maybe show only "Downloading" or "Installing" based on which phase is active?